eth/backend.go

```diff
@@ -134,7 +134,10 @@ type Ethereum struct {
 	// When we receive something here, it means that the beacon chain transitioned
 	// to proof-of-stake so we start reverse syncing from the header
 	reverseDownloadCh     chan privateapi.PayloadMessage
-	waitingForBeaconChain uint32 // atomic boolean flag
+	unwindForkChoicePOSCh chan uint64
+	unwindFinished        chan bool
+	waitingForBeaconChain uint32    // atomic boolean flag
+	engineCond            sync.Cond // Cond that comunicates to engine API when to unlock if it is waiting
```

Comment:
There's no need for this variable since syncCond is only used internally by `EthBackendServer`. Please remove the variable and restore the original logic where `NewEthBackendServer` created its own syncCond.